### DISCUSSION TITLE
Possible Mojave support and audio channels

### DISCUSSION THREAD

**Discussion by Robmanzzz** [sbarex/MediaInfo#9] [Ideas]
Hi!

I would love to install your app but I'm still on Mojave, is it possible to add support for 10.14? And could you add the ability to see wether an audio file is mono, stereo, multichannel?

Thanks for answering!

Rob

**Comment by sbarex** [comment]
I added the support for audio channels on release 1.5.4.

I don't have a macos 10.14 version to test if the application works. However now I have recompiled it for 10.14 (unfortunately it weighs more because it has to incorporate the swift libraries that are available in the system only from 10.15 upwards). 

Please check if it works.

**Comment by Robmanzzz** [comment]
Hi!

Wow, thanks for the lightning quick response! But I'm afraid this version seems to be damaged...
![Screenshot 2021-06-01 at 19 46 52](https://user-images.githubusercontent.com/8589665/120368625-931e5b80-c312-11eb-8960-acd5a8c13e7a.png)

**Comment by sbarex** [comment]
It is due to the quarantine system and the fact that the application is unsigned. 

Use this terminal command:

```sh
 xattr -cr /Applications/MediaInfoEx.app
```

(replace _/Applications/MediaInfoEx.app_ with the full path of the application)

**Reply by Robmanzzz** [reply]
Should've read the read me first, sorry. So far it looks like it works like it should. Thank you so much! Can I donate to you somewhere?

**Reply by sbarex** [reply]
Thanks, more than one has suggested me to publish my applications on the store, but they often don't follow Apple's exact guidelines and probably wouldn't be accepted, and at the moment I haven't found a viable solution to receive donations.

**Comment by Robmanzzz** [comment]
And do you think it's possible to implement it like shown in attached mockup?
![Screenshot 2021-06-01 at 20 48 08](https://user-images.githubusercontent.com/8589665/120375329-f57b5a00-c31a-11eb-9968-56fd0a833725.png)

**Reply by sbarex** [reply]
Unfortunately Apple doesn't allow this kind of Finder customization.
